package.json

```diff
@@ -99,12 +99,12 @@
     "build": "buble src --output build",
     "clean": "node ./bin/clean.js",
     "gh-release": "gh-release",
-    "integration-test": "npm run build && node ./test",
     "open-config": "node ./bin/open-config.js",
     "package": "node ./bin/package.js",
     "prepublish": "npm run build",
     "start": "npm run build && electron .",
     "test": "standard && depcheck --ignores=buble,lodash.merge,nodemon,gh-release --ignore-dirs=build,dist && node ./bin/extra-lint.js",
+    "test-integration": "npm run build && node ./test",
```

Comment:
You'll need to update the [readme note](https://github.com/feross/webtorrent-desktop#run-integration-tests) too.